frontend/awx/resources/inventories/components/RunCommandSteps.tsx

```diff
@@ -0,0 +1,145 @@
+import { useTranslation } from 'react-i18next';
+import { PageFormSection } from '../../../../../framework/PageForm/Utils/PageFormSection';
+import {
+  PageDetail,
+  PageDetails,
+  PageFormCheckbox,
+  PageFormDataEditor,
+  PageFormSelect,
+  PageFormSwitch,
+  PageFormTextInput,
+} from '../../../../../framework';
+import { PageFormCredentialSelect } from '../../../access/credentials/components/PageFormCredentialSelect';
+import { usePageWizard } from '../../../../../framework/PageWizard/PageWizardProvider';
+import { PageDetailCodeEditor } from '../../../../../framework/PageDetails/PageDetailCodeEditor';
+import { RunCommandWizard } from '../../../interfaces/Inventory';
+import { LabelGroup } from '@patternfly/react-core';
+import { CredentialLabel } from '../../../common/CredentialLabel';
+
+export function RunCommandDetailStep() {
+  const { t } = useTranslation();
+  const moduleOptions = [
+    { label: t('command'), value: 'command' },
+    { label: t('shell'), value: 'shell' },
+    { label: t('yum'), value: 'yum' },
+    { label: t('apt'), value: 'apt' },
+    { label: t('apt_key'), value: 'apt_key' },
+    { label: t('apt_repository'), value: 'apt_repository' },
+    { label: t('apt_rpm'), value: 'apt_rpm' },
+    { label: t('service'), value: 'service' },
+    { label: t('group'), value: 'group' },
+    { label: t('user'), value: 'user' },
+    { label: t('mount'), value: 'mount' },
+    { label: t('ping'), value: 'ping' },
+    { label: t('selinux'), value: 'selinux' },
+    { label: t('setup'), value: 'setup' },
+    { label: t('win_ping'), value: 'win_ping' },
+    { label: t('win_service'), value: 'win_service' },
+    { label: t('win_updates'), value: 'win_updates' },
+    { label: t('win_group'), value: 'win_group' },
+    { label: t('win_user'), value: 'win_user' },
+  ];
+
+  const verbosityOptions = [
+    { label: t('0 (Normal)'), value: '0' },
+    { label: t('1 (Verbose)'), value: '1' },
+    { label: t('2 (More Verbose)'), value: '2' },
+    { label: t('3 (Debug)'), value: '3' },
+    { label: t('4 (Connection Debug)'), value: '4' },
+    { label: t('5 (WinRM Debug)'), value: '5' },
+  ];
+
+  return (
+    <PageFormSection singleColumn>
```

Comment:
Missing labelHelps for the form. Please include them for all fields and also for other forms in wizard.
![image](https://github.com/ansible/ansible-ui/assets/23277791/8f4fd71d-4fc2-4363-b83b-baa24ef65329)